<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<?pydev-testrun version="1.0"?><pydev-testsuite>
<summary errors="1" failures="0" finished="true" name="Test Run: 0" tests="0" total_time="Finished in: 0.00 secs."/>
<test location="C:\Users\ITAUser\Desktop\python\Logical Operators\debugLogicalOperatorDrills.py" status="error" test="debugLogicalOperatorDrills" time="0">
<stdout/>
<stderr><![CDATA[Traceback (most recent call last):
  File "C:\Program Files\Brainwy\LiClipse 6.0.0\plugins\org.python.pydev.core_7.4.0.201910251202\pysrc\_pydev_runfiles\pydev_runfiles.py", line 468, in __get_module_from_str
    mod = __import__(modname)
  File "C:\Users\ITAUser\Desktop\python\Logical Operators\debugLogicalOperatorDrills.py", line 47, in <module>
    not(a == 3)
NameError: name 'a' is not defined
ERROR: Module: debugLogicalOperatorDrills could not be imported (file: C:\Users\ITAUser\Desktop\python\Logical Operators\debugLogicalOperatorDrills.py).
]]></stderr>
</test>
<launch mode="debug">
<launch_memento><![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<launchConfiguration local="true" path="Logical Operators debugLogicalOperatorDrills.py"/>
]]></launch_memento>
</launch>
</pydev-testsuite>
